src/server/qgsserversettings.h

```diff
@@ -77,6 +77,8 @@ class SERVER_EXPORT QgsServerSettingsEnv : public QObject
       QGIS_SERVER_WCS_SERVICE_URL, //!< To set the WCS service URL if it's not present in the project. (since QGIS 3.20).
       QGIS_SERVER_WMTS_SERVICE_URL, //!< To set the WMTS service URL if it's not present in the project. (since QGIS 3.20).
       QGIS_SERVER_LANDING_PAGE_PREFIX, //! Prefix of the path component of the landing page base URL, default is empty (since QGIS 3.20).
+      QGIS_SERVER_CONFIG_CACHE_CHECK_INTERVAL, //! Set the interval for cache invalidation check, default to 0 which select the legacy File system watcher  (since QGIS 3.24).
+      QGIS_SERVER_DISABLE_CONFIG_CACHE, //! Disable internal config cache  (since QGIS 3.24).
```

Comment:
```suggestion
      QGIS_SERVER_PROJECT_CACHE_STRATEGY, //! 'filesystem', 'interval' or 'off' \since QGIS 3.24
```

Suggestion to
- control the behavior mainly via this (string/enum based)
- only use the interval configuration if this is set here ( no special value / negative ... handling)
- use the term `project cache` rather than 'config cache', I think that's more precise as term

Reply:
I would use `periodic` instead of `interval`.

Reply:
@m-kuhn @elpaso wouldn't it be better to separate the two strategies in two QgsConfigCache sub-classes ?

Reply:
Or separate the two strategies in different classes...